### PR TITLE
Bpr 92 class coupling

### PR DESCRIPTION
**Pull request by stefaniatomuta**
Command grouped based on name to return list of values for class coupling:

image
Not sure about the inheritanceCommand, I commented for now the max_depth as it screams 

This is what was before as dict:
image

### CODE REVIEW

File: Service/FileLoadService.py [hunk 1]
```diff
@@ -28,24 +26,35 @@ def process_data_from_folder():
     commands = get_commands()
     folder = fd.askdirectory()
     sums = {}
-    class_matches = []
+    dict_matches = {}
     csfiles = 0
-    for root,dirs,files in os.walk(folder):
+    # for entry in os.scandir(folder):
```
Comment (HolaSombreros) [inline Service/FileLoadService.py:31]: if this is commented then do we actually need it?

File: Service/FileLoadService.py [hunk 2]
```diff
@@ -28,24 +26,35 @@ def process_data_from_folder():
...
             for command in commands:
-                file_path = os.path.join(root,file_name)
+                command_name = type(command).__name__
+                file_path = os.path.join(root, file_name)
                 analysis_results = command.execute(file_path)
-                for file, match in analysis_results.items():
-                    info = FrameworksService.EOL_API(match)
-                    print(info.isEndOfLife)
                 if(isinstance(analysis_results, list)):
-                    class_matches.extend(analysis_results)
```
Comment (HolaSombreros) [inline Service/FileLoadService.py:44]: what is the reason for this change? this is probably why the inheritance tree doesn't work

File: Service/ModuleCouplingService.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import re
+from Helpers.RegexHelper import *
+
+def test(file_name) -> list:
```
Comment (HolaSombreros) [inline Service/ModuleCouplingService.py:4]: i think it would be better to change the name of this method so that it is more suggestive

File: Service/FileLoadService.py (hunk 2 above)
Comment (HolaSombreros) [reply Service/FileLoadService.py:44]: ok I can see that it is because of the metrics command that has the same return type. it seems that the list these two commands return are different so we need to handle their response somehow

Comment (stefaniatomuta) [reply Service/FileLoadService.py:44]: Yes, we need to find a solution for this

Comment (HolaSombreros) [reply Service/FileLoadService.py:44]: I'll look into it. For now we can leave it like this

**Review by HolaSombreros — APPROVED** [review]
